app/components/global/layout/DisplayBasedOnScreenSize.js

```diff
@@ -0,0 +1,13 @@
+import styled from 'styled-components'
+
+export const HideOnMobile = styled.div`
+  @media (max-width: 480px) {
```

Comment:
Did you copy and paste `${th('breakpoints.0)}` because if so, you're missing a `'`.

Reply:
It's odd as the media query is correctly showing in the inspector. The browser is just ignoring it 🤔

Reply:
@tamlyn haha, no I didn't

@Cooryd Do you mean if we put in 'breakpoints.0', it's showing in the inspector? How weird

Reply:
Yes, I can see the 480px within the browser inspector so it seems to be generating the class correctly but the browser isn’t rendering the query like you said.

Reply:
Oh. The breakpoint is `480` without units. Need to add `px`.
```suggestion
  @media (max-width: ${th('breakpoints.0')}px) {
```

(Trying out new GitHub code suggestion thingie ^^^)

Reply:
Ooo I like the suggestion thingy. But I'd like to make a commit that changes both '480px' to 'breakpoints' at once :thinking:

Reply:
https://www.styled-components.com/docs/advanced#media-templates details using a a helper function for media queries which might be useful. There example fixes it as a max-width though, theres also [styled-components-breakpoint](https://github.com/wearehumblebee/styled-components-breakpoint) which takes this an adds more functionality so you can set breakpoint ranges

Reply:
Ah that's a great shout. Thanks @Cooryd!

I originally had the media queries directly on the `Box` & `StickyFooter` components that I want to hide. But then extracted them into generic wrapper components, so that we could reuse these queries elsewhere. But adding an extra, unnecessary bit of HTML to the page is definitely not ideal. Creating a helper function is much better. Cheers! :+1: